### ISSUE TITLE
Можно ли получить котировки по фьючерсам на ММВБ?

### ISSUE THREAD

**Issue by mrals2018** [WLM1ke/apimoex#3]
Приветствую! Подскажите, а можно ли с помощью вашего решения выкачать котировки фьючерса, например, на индекс RTS или BRENT? Просто уже всё перепробовал, но ничего не получается...

**Comment by WLM1ke** [comment]
Думаю можно, но я в фьючерсная мало понимаю. 
У них какие-то стандартные тиккер есть, по аналогии с акциями — напишите. 
Графики по ним можно поглядеть на сайте биржи — скиньте ссылку.

**Comment by mrals2018** [comment]
> Думаю можно, но я в фьючерсная мало понимаю.
> У них какие-то стандартные тиккер есть, по аналогии с акциями — напишите.
> Графики по ним можно поглядеть на сайте биржи — скиньте ссылку.

Да, у них есть тикеры, которые меняются раз в месяц или раз в несколько месяцев. Для фьючерса на индекс RTS (который текущий) вот ссылка https://www.moex.com/ru/contract.aspx?code=RTS-3.21. 
На TradingView все фьючерсы на индекс RTS или на BRENT объеденены в RI1! и BR1! соответственно.
Для BRENT вот ссылка https://www.moex.com/ru/contract.aspx?code=BR-3.21

**Comment by WLM1ke** [comment]
```
with requests.Session() as session:
    data = apimoex.get_market_candles(
        session=session,
        security="RIH1",
        market="forts",
        engine="futures",
        interval=24,
    )
    df = pd.DataFrame(data)
    df.set_index("begin", inplace=True)
    print(df.head(), "\n")
    print(df.tail(), "\n")
    df.info()
```

**Comment by WLM1ke** [comment]
У нефти тикер "BRH1", насколько я понял, соответсвенно нужно security="BRH1" поставить.

**Comment by mrals2018** [comment]
Блин! Точно! Всё работает! Прошу прощения! )) В слове futures опечатался... Охххх.. Рука-лицо! Ещё раз извиняюсь и ОГРОМНОЕ СПАСИБО! ))
